src/scripts/main.js

```diff
@@ -1,3 +1,54 @@
 'use strict';
 
 // write code here
+
+const employees = document.querySelectorAll('li');
+const listEmployees = document.querySelector('ul');
+
+function getSalary(element) {
+  const salary = element.dataset.salary;
+
+  if (salary) {
+    let cleanedSalary = salary.slice(1);
+
+    cleanedSalary = cleanedSalary.split(',').join('');
+
+    return Number(cleanedSalary);
+  }
+
+  return 0;
```

Comment:
The `getSalary` function will return 0 if the salary is not present or if it is not a valid number. This could potentially hide errors if the salary data is not formatted correctly. It may be better to throw an error in these cases, so that the problem can be detected and fixed.